tools/netlist/CMakeLists.txt

```diff
@@ -3,7 +3,8 @@
 # SPDX-License-Identifier: MIT
 # ~~~
 
-add_executable(slang_netlist netlist.cpp source/Netlist.cpp)
+add_executable(slang_netlist netlist.cpp source/Netlist.cpp
+                             source/CombLoops.cpp)
```

Comment:
It would be better to include this source in the netlist library, so that the functionality is not only bound to the command-line tool.

Reply:
Ignore me, I thought that I had set up a library!